clusters/grid/umd4/repository/config.pan

```diff
@@ -0,0 +1,56 @@
+# NOTE: This template should be the LAST thing included in a
+# machine profile.  If you include packages after this template
+# then they will not be "resolved" and SPMA will not function
+# correctly.
+
+unique template repository/config;
+
+include'pan/functions';
+
+include 'repository/snapshot/snapshot_variables';
+
+# Repositories related to base OS and quattor client (should be first)
+include'repository/config/os';
+
+# Quattor repositories
+include 'repository/config/quattor';
+
+# Local Repositories
+variable YUM_SITE_SNAPSHOT_NS ?= YUM_SNAPSHOT_NS;
+variable SITE_REPOSITORY_LIST ?= list();
+variable SITE_REPOSITORY_CONFIG ?= dict();
+variable DEBUG = debug(
+    'OS_REPOSITORY_LIST = ' + to_string(OS_REPOSITORY_LIST) + "\n" +
+    'SITE_REPOSITORY_LIST = ' + to_string(SITE_REPOSITORY_LIST) + "\n" +
+    'SITE_REPOSITORY_CONFIG = ' + to_string(SITE_REPOSITORY_CONFIG) + "\n"
+);
+include'quattor/functions/repository';
+#'/software/repositories' = add_repositories(SITE_REPOSITORY_LIST,YUM_SITE_SNAPSHOT_NS);
+'/software/repositories' = add_repositories(SITE_REPOSITORY_LIST);
+'/software/repositories' = repository_config(SITE_REPOSITORY_CONFIG);
+
+# Repositories related to grid middleware
+# Only if it can be found in the loadpath (else this is a machine not running gLite
+include if_exists('repository/config/grid');
+
+# Repositories related to Nagios
+variable REPOSITORY_CONFIG_NAGIOS ?= null;
+variable REPOSITORY_CONFIG_NAGIOS_INCLUDE =
+if ( exists(REPOSITORY_CONFIG_NAGIOS) && is_defined(REPOSITORY_CONFIG_NAGIOS) ) {
+    REPOSITORY_CONFIG_NAGIOS;
+} else {
+    null;
+};
+include REPOSITORY_CONFIG_NAGIOS_INCLUDE;
```

Comment:
This is strangely over-engineered.
```suggestion
include if (is_defined(REPOSITORY_CONFIG_NAGIOS)) if_exists(REPOSITORY_CONFIG_NAGIOS);
```

Reply:
I was so focused on the indentation problem that I missed the simplifaction required. I'll fix it!